spec/models/course_user_spec.rb

```diff
@@ -235,6 +235,19 @@
       end
     end
 
+    describe '#last_obtained_achievement' do
```

Comment:
preferably, you should make the first commit just deal with models first, that way you know what you need to test.